code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -307,9 +307,10 @@ emp_act
 				if(prob(final_force))
 					if(apply_effect(20, PARALYZE, armor))
 						visible_message("<span class='danger'>[src] has been knocked unconscious!</span>")
-						// if(src != user && I.damtype == BRUTE && isrev(src))
-						// 	ticker.mode.remove_revolutionary(mind)
-						// 	add_attacklogs(user, src, "de-converted from Revolutionary!")
+						var/datum/role/revolutionary/R = mind.GetRole(REV)
+						if(R && istype(R) && stat != DEAD && src != user && I.damtype == BRUTE)
+							R.Drop()
```

Comment:
this also works on revheads which it probably shouldn't? or should it?

Reply:
There's a difference between REV and REVHEAD

Reply:
It doesn't de-convert headrevs.
`mind.GetRole(REV)` will return 0 unless they are a normal revolutionary.